src/cli/cli_br.cpp

```diff
@@ -568,6 +568,50 @@ template <> otError Br::Process<Cmd("nat64prefix")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli br nat64prefixtable
+ * @code
+ * br nat64prefixtable
+ * prefix:fd00:1234:5678:0:0:0::/96, ms-since-rx:29526, lifetime:1800, router:ff02:0:0:0:0:0:0:1 (M:0 O:0 S:1)
```

Comment:
`ff02:0:0:0:0:0:0:1` is Link local multicast, not a valid unicast Link Local address for discovered routers